docs/component-testing.md

```diff
@@ -0,0 +1,138 @@
+## Testplane Component Testing (experimental)
+
+Almost all modern web interfaces are written using frameworks (React, Vue, Svelte, ...) to simplify the creation, reuse and composition of web components. It is important to test such components in isolation from each other to be sure that each component is doing its job correctly. Just like we write unit tests separately from integration tests. Testplane already supports testing components using [Storybook](https://storybook.js.org/) (via [@testplane/storybook](https://github.com/gemini-testing/testplane-storybook) plugin), but this tool is not relevant for all projects. Therefore, we have developed another component testing option that does not require the use of Storybook.
```

Comment:
👍 for "testplane-storybook" advertising

> Almost all modern web interfaces are written

I think more english would be "Almost every modern web interface is written..."

Reply:
> to be sure that each component

"that" seems to be extra

Reply:
> I think more english would be "Almost every modern web interface is written..."

Hm, to my mind, my version is better